update.sh

```diff
@@ -5,9 +5,10 @@ cd "$(dirname "$(readlink -f "$BASH_SOURCE")")"
 
 versions=( */ )
 versions=( "${versions[@]%/}" )
-downloadable=$(curl -sSL 'http://www.elasticsearch.org/downloads' | sed -rn 's!.*?http://www.elasticsearch.org/downloads/[0-9]+-[0-9]+-[0-9]+/">Download v ([0-9]+\.[0-9]+\.[0-9]+)<.*!\1!gp')
+downloadable=$(curl -sSL 'https://www.elastic.co/downloads/past-releases' | gsed -rn 's!.*?/downloads/past-releases/[0-9]+-[0-9]+-[0-9]+">Elasticsearch ([0-9]+\.[0-9]+\.[0-9]+)<.*!\1!gp')
```

Comment:
I don't think `gsed` exists on most Linux systems. As much as it sucks for me personally to have trouble running the `*.sh` scripts in these repositories under OS X, I stay away from trying to make them cross-platform. The one that regularly bites me is the lack of `readlink -f`, so I generally run `update.sh` and `generate-stackbrew-library.sh` under a `buildpack-deps` container in Docker :wink: